### DISCUSSION TITLE
Ubuntu 18 still missing

### DISCUSSION THREAD

**Discussion by Nardol** [BearWare/TeamTalk5#1705] [General]
Hello @bear101
Opening Teamtalk, a new beta version dialog is displayed.
And I don't find anything for Ubuntu 18, which is and will always be the only available version for Debian Buster and also Bullseye which is the stable version for this last. Ubuntu 22 depend on LibreSSL, Debian depends on OpenSSL so it is impossible to use Ubuntu 22 version with another distro than Ubuntu 22 without hack.
Same situation for TeamTalk pro server.
I don't use SentOS but all SentOS versions are also missing.

Will Debian users have to compile to be able to test TeamTalk forever?
In my point of view, using TeamTalk without WebRTC is not acceptable, that's why I don't use Github actions artifacts.

Please, could it be possible to have Ubuntu 18 version available?

**Comment by bear101** [comment]
I'll make the final release available for Ubuntu 18. Beta version are only compiled for a limited number of platforms.

**Comment by so-demonio** [comment]
Hello, today is March 2. Is it possible to install latest TeamTalk version to Ubuntu 18 or 20? Because I don't want to use ubuntu 22.04. It has some handycups. If it's possible, please someone help me to do it.

**Reply by lbk2907** [reply]
Hey there, check out this archive.
https://bearware.dk/teamtalk/
